### ISSUE TITLE
Flask app when simple quote is in denomination on search endpoint

### ISSUE THREAD

**Issue by phe-sto** [phe-sto/enthic#21]
See the attached test report, all searches crashes.

[Test Results — pytest in test_app.py.pdf](https://github.com/phe-sto/enthic/files/6255420/Test.Results.pytest.in.test_app.py.pdf)

**Comment by leonarf** [comment]
Fix is available in PR #17

**Comment by leonarf** [comment]
Fix available in PR #31

**Comment by phe-sto** [comment]
Hi, geat job @leonarf 
I think with can review this later but please keep in mind the compute annual statistic tests failed:
```
FAILED [ 38%]
AssertionError: WRONG HTTP RETURN CODE 500 INSTEAD OF 200
500 != 200
Expected :
200
Actual :
500
host = '127.0.0.1:5000', first_letters = 'aa'
@pytest.mark.parametrize("first_letters", LETTERS_COUPLE_LIST)
def test_compute_annual_statistics(host, first_letters):
"""
Test the /compute/ endpoint. Should return a JSON and RC 200.

:param host: Fixture of the API host.
:param first_letters: Filter to compute only on companies whose denomination begin with these letters.
"""
response = get("http://" + host + "/compute/" + first_letters)
>       assert response.status_code == 200, "WRONG HTTP RETURN CODE %s INSTEAD OF 200" % response.status_code
E AssertionError: WRONG HTTP RETURN CODE 500 INSTEAD OF 200
E assert 500 == 200
test_app.py:438: AssertionError
```

**Comment by phe-sto** [comment]
Indeed fixed in PR Fix available in PR #31
